misc/multiaddr/src/protocol.rs

```diff
@@ -180,7 +180,7 @@ impl<'a> Protocol<'a> {
                 let mut rdr = Cursor::new(data);
                 let mut seg = [0_u16; 8];
 
-                for x in seg.iter_mut() {
+                for x in se.g.iter_mut() {
```

Comment:
What happened here?

Reply:
Whoops, I thought I did it with word breaks, I'll try again.